modules/tensor_mechanics/include/userobjects/VectorPostprocessorToInterpolator.h

```diff
@@ -0,0 +1,60 @@
+//* This file is part of the MOOSE framework
```

Comment:
I think I would rename this to something like 'CSVInterpolator' or 'InterpolateFromCSV'. The CSV files happen to be generated by a VectorPostprocessor in your case, but they could really come from anywhere.